### PR TITLE
Wasmtime: remaps for coverage

### PR DESCRIPTION
**Pull request by catenacyber**
cc @alexcrichton cf https://github.com/google/oss-fuzz/pull/5366#issuecomment-799476131

This should fix wasmtime coverage build

### CODE REVIEW

File: projects/wasmtime/build.sh [hunk 1]
```diff
@@ -26,6 +26,12 @@ build() {
   shift
   PROJECT_DIR=$SRC/$project
 
+  cd $PROJECT_DIR
```
Comment (alexcrichton) [inline projects/wasmtime/build.sh:29]: Could this have a comment for the purpose of what's happening? (basically just ensuring that absolute paths are used everywhere in debuginfo and such)

**Review by alexcrichton — APPROVED** [review]
I always find bash a bit, uh, esoteric to read...

In any case though looks good to me, thanks!

**Review by alexcrichton — APPROVED** [review]
Thanks for your help here!

**Review by inferno-chromium — APPROVED** [review]

**Review by inferno-chromium — APPROVED** [review]

**Comment by inferno-chromium** [comment]
> @inferno-chromium could we merge this ?

Can you check CI failures, are they related to this change, can you maybe push again to see if they still happen. if not, leave a comment and we can merge.

**Comment by catenacyber** [comment]
CI is happier except for coverage failure which seems unrelated
`LLVM ERROR: IO failure on output stream: No space left on device`

**Comment by alexcrichton** [comment]
Thanks again for this @catenacyber, we got our [first successful build today](https://oss-fuzz-build-logs.storage.googleapis.com/log-6eaef32e-71a1-4b24-8763-8ccab0dff833.txt) as a result.

@inferno-chromium if I go to https://oss-fuzz.com/ and click "total coverage" it links me to https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_wasmtime/latest which gives me " Failed to get coverage report. (alex@alexcrichton.com) ". Is that something that needs to be configured on y'all's end? Or should I just wait a day or so for things to propagate?

**Comment by catenacyber** [comment]
There is one available here
https://storage.googleapis.com/oss-fuzz-coverage/wasmtime/reports/20210322/linux/report.html

**Comment by alexcrichton** [comment]
Oh wow, thanks! How did you end up finding the link to that?

**Comment by catenacyber** [comment]
Just clicked a link to one of my projects and replaced the name ;-)
I guess you can wait a day or two for thing to propagate...

**Comment by catenacyber** [comment]
And now your link https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_wasmtime/latest works (and redirects to mine)

**Pull request merged**